<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1898</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="ax1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>491</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="ax2" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>491</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="ax3" native="true">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>0</y>
      <width>491</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="GLViewWidget" name="ax4" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>491</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="ax5" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>491</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="ax6" native="true">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>440</y>
      <width>491</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="sINRetardance">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>80</y>
      <width>20</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sINAngle">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>80</y>
      <width>20</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sDUTPDL">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>360</y>
      <width>20</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sDUTRetardance">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>360</y>
      <width>20</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>670</y>
      <width>84</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="sINRetardance_label">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>L1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sINAngle_label">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>L2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sDUTPDL_label">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>320</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>L3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sDUTRetardance_label">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>320</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>L3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sINRetardance_value">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sINAngle_value">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sDUTPDL_value">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>340</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sDUTRetardance_value">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>340</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="sDUTAngle">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>360</y>
      <width>20</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sINRotAngle">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>80</y>
      <width>20</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sINRotAngle_value">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sINRotAngle_label">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>L2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sDUTAngle_label">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>320</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>L3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sDUTAngle_value">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>340</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>670</y>
      <width>84</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1898</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLViewWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.opengl</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
